ner_extended/fetch_assets.py

```diff
@@ -0,0 +1,33 @@
+import spacy
+from spacy.tokens import DocBin, Doc
+from datasets import load_dataset
+
+
+def fetch_dansk():
+    # DANSK has yet to publically released. Write the authors for a request of early read access.
+
+    # Download the datasetdict from the HuggingFace Hub
+    try:
+        datasets = load_dataset("chcaa/DANSK")
+    except FileNotFoundError:
+        # print(
+        #     "\nERROR: DANSK has yet to publically released. Write the authors for a request of early read access. Shutting down.\n",
+        # )
+        raise FileNotFoundError(
+            "DANSK has yet to publically released. Write the authors for a request of early read access. \n\nShutting down.",
+        )
```

Comment:
```suggestion
        raise FileNotFoundError(
            "DANSK is not available. It might be due a HF being down on the dataset not yet being publically released.",
        )
```